src/screens/DiscoveryScreenV2/index.js

```diff
@@ -20,11 +21,10 @@ import {
   DISCOVERY_TAB_TOPICS,
   DISCOVERY_TAB_USERS
 } from '../../utils/constants';
+import {Header} from '../../components';
 import {colors} from '../../utils/colors';
 import {fonts} from '../../utils/fonts';
 import {withInteractionsManagedNoStatusBar} from '../../components/WithInteractionManaged';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, the import order has been changed. While this doesn't affect the functionality of the code, it's generally a good practice to maintain a consistent order of imports for better readability and maintainability. 

```diff
- import {colors} from '../../utils/colors';
- import {fonts} from '../../utils/fonts';
- import {withInteractionsManagedNoStatusBar} from '../../components/WithInteractionManaged';
- import FollowingAction from '../../context/actions/following';
- import {Header} from '../../components';
+ import {Header} from '../../components';
+ import {colors} from '../../utils/colors';
+ import {fonts} from '../../utils/fonts';
+ import {withInteractionsManagedNoStatusBar} from '../../components/WithInteractionManaged';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->